src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -73,8 +108,15 @@ export function useSwapInputsController({
   });
   const incrementDecimalPlaces = useDerivedValue(() => countDecimalPlaces(niceIncrement.value));
 
+  const inputNativePrice = useDerivedValue(() => {
+    return internalSelectedInputAsset.value?.nativePrice || internalSelectedInputAsset.value?.price?.value || 0;
```

Comment:
note to self: remind me the difference between these sources of prices?

Reply:
nativePrice gets set via the fetchAssetPrices interval / calls.
asset price.value can be set if we previously had the price from it being in the userAssets (aka those assets include price data)